content/client-server-api/modules/reference_relations.md

```diff
@@ -0,0 +1,51 @@
+---
+type: module
+---
+
+### Reference relations
+
+{{% added-in v="1.4" %}}
+
+Generically referencing another event can be done with a `rel_type` of `m.reference`
+as a form of [relationship](#forming-relationships-between-events). There is no
+implied meaning behind the reference, and is usually context-dependent. Such an
+example is the [key verification framework](#key-verification-framework) which uses
+reference relations to associate distinct events with a specific verification attempt.
+
+The aggregation format of `m.reference` relations consists of a single `chunk` property to
+indicate all the events which `m.reference` the event (the parent). Currently,
+only a single `event_id` field is present on the events in the `chunk`.
+
+{{% boxes/note %}}
+Clients which wish to use threads or replies are expected to use other relationship
+types than references. References are typically used to associate data rather than
+messages.
+{{% /boxes/note %}}
+
+An example `m.reference` would be:
+
+```json
+{
+  "content": {
+    "m.relates_to": {
+      "rel_type": "m.reference",
+      "event_id": "$another_event"
+    }
+    // other content fields as required
+  }
+  // other fields as required by events
+}
+```
+
+The [bundle](#aggregations) under `m.relations` would appear similar to the following:
```

Comment:
s/bundle/aggregation/ ?

Reply:
I think this is a bundle, in reference to the same language `m.replace` uses.